infrastructure/src/main/java/org/corfudb/infrastructure/LogUnitServer.java

```diff
@@ -594,27 +594,25 @@ public static LogUnitServerConfig parse(Map<String, Object> opts) {
      * This facilitates the injection of mocked objects during unit tests.
      */
     public static class LogUnitServerInitializer {
-        StreamLog buildInMemoryStreamLog() {
+        public StreamLog buildInMemoryStreamLog() {
             return new InMemoryStreamLog();
         }
 
-        StreamLog buildStreamLog(@Nonnull LogUnitServerConfig config,
-                                 @Nonnull ServerContext serverContext) {
-            return new StreamLogFiles(serverContext, config.isNoVerify());
+        public StreamLog buildStreamLog(@Nonnull ServerContext serverContext) {
+            return new StreamLogFiles(serverContext);
         }
 
-        LogUnitServerCache buildLogUnitServerCache(@Nonnull LogUnitServerConfig config,
-                                                   @Nonnull StreamLog streamLog) {
-            return new LogUnitServerCache(config, streamLog);
+        public LogUnitServerCache buildLogUnitServerCache(@Nonnull StreamLog streamLog,
```

Comment:
This change was to fix the static codacy issues, which didn’t like the package private default scope.

Reply:
That's a good idea, since they were package-private before. However, making them protected would prevent any IT's from using these methods. Do we know that these methods won't be used in IT's in the future?

Reply:
I don't believe so. I had added this class to facilitate the injection of mocks in unit tests.